packages/support/resources/lang/en/actions/create.php

```diff
@@ -4,7 +4,7 @@
 
     'single' => [
 
-        'label' => 'New :label',
```

Comment:
I prefer the old one for this action, since its usually on its own and not associated with a particular record like "Edit" is, so its clear what you are creating

Also `New` instead of `Create` since `Create` is the action of saving

Reply:
I understand your thinking. However, this is a little harder in many languages (like Dutch) where the word `New` should be inflected based on the word passed to `:label`.

So for example:
- Nieuwe reactie
- Nieuw bericht

And as I already saw this key being translated as just `Create`, I thought it would be good to make this consistent across locales.

Reply:
I'm fine with it being different across locales if that works best, but in English it's clearer to use `New :label` IMO

Reply:
Makes sense to use the best possible option for every language!